### DISCUSSION TITLE
Add citation list

### DISCUSSION THREAD

**Discussion by erlebach** [Hugo-COLLIN/SaveMyPhind-conversation-exporter#42] [Ideas]
Perplexity only uses a subset of the citations it searches. It would be nice to optionally include a list of all the citations (which are often quite good), and make them clickable). They could be added to the end of the markdown file to avoid being distracted. I would also suggest writing the citations as [1] [2], rather than 1 2. That would make them more visible.

**Comment by Hugo-COLLIN** [comment]
@erlebach Thank you, I will take into account your suggestions.

If not already done, you can complete the following questionnaire so I can understand better your expectations with Save my Chatbot: [https://save.hugocollin.com/feedback](https://save.hugocollin.com/feedback).
